src/KeyboardHandlerDropdown/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import "../common/variables";
+
+.keyboard-handler-dropdown {
```

Comment:
can remove this when you remove label since there won't be a need for the wrapper div

Reply:
renamed `.keyborad-handler-dropdown__select` -> `.keyborad-handler-dropdown`